src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail_controller.js

```diff
@@ -108,6 +106,18 @@ export default class ReplicationControllerDetailController {
      * @export {boolean}
      */
     this.eventsOrder = DOWNWARDS;
+
+    /**
+     * If true then sidebar will be displayed, otherwise it won't.
+     * @export {boolean}
+     */
+    this.displaySidebar = this.mdMedia('gt-sm');
+
+    /**
+     * Binding resize listener, which switches displaySidebar boolean variable.
+     */
+    angular.element(this.window)
```

Comment:
Could you explain a little bit more, because I don't understand? Are you saing, that `this.displaySidebar = this.mdMedia('gt-sm');` and `angular.element(this.window).bind('resize', () => { this.displaySidebar = this.mdMedia('gt-sm');});` should be moved to directive?